src/components/form-elements/draft-js-mention-selector/createMentionSelectorState.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import { ContentState, EditorState, Modifier, SelectionState } from 'draft-js';
+import DraftMentionDecorator from './DraftMentionDecorator';
+
+// returns data for first mention in a string
+const getMentionFromText = (text: string) => {
+    // RegEx.exec() is stateful, so we create a new regex instance each time
+    const MENTION_REGEX = /([@＠﹫])\[([0-9]+):([^\]]+)]/gi;
```

Comment:
It seems like we should use camelCase rather all-caps here, since we are intentionally reintantiating this regex.